client/src/actions/annotation.ts

```diff
@@ -1,24 +1,22 @@
-/*
-Action creators for user annotation
-*/
+/**
+ * Action creators for user annotation
+ */
 import difference from "lodash.difference";
 import pako from "pako";
 import * as globals from "../globals";
+import { AppDispatch, GetState } from "../reducers";
 import { MatrixFBS, AnnotationsHelpers } from "../util/stateManager";
 
 const { isUserAnnotation } = AnnotationsHelpers;
 
+/**
+ * Add a new user-created category to the obs annotations.
+ * @arg newCategoryName - string name for the category.
```

Comment:
should be `@param` and not `@arg`

See https://tsdoc.org/pages/tags/param/

Reply:
Also see https://tsdoc.org/pages/tags/returns/

Reply:
Ahh only jsdoc has `@arg` alias

Reply:
This function returns `Promise<void>`, so I suppose we don't need `@returns`?

Reply:
It is certainly optional, but my _personal opinion_ is that being explicit is very helpful - so I bias towards always specifying `@returns`.

Reply:
Ahh yeah that sounds good 👍 Just added `@returns An empty promise.`!